requirements/horovod-gpu-environment.yml

```diff
@@ -0,0 +1,23 @@
+name: horovod-gpu
+
+channels:
+  - nvidia
+  - conda-forge
+  - defaults
+
+dependencies:
+  - ccache
+  - cmake=3.22
```

Comment:
this environment will add 1+ gb every time it runs(on every test execution for models repo, every commit). This is probably not what we want.